pkg/cache/clusterqueue.go

```diff
@@ -694,7 +707,7 @@ func (c *ClusterQueue) DominantResourceShare(w *workload.Info) (int, corev1.Reso
 	for rName, rStats := range c.ResourceStats {
 		var ratio int64
 		if c.Cohort.ResourceStats[rName].Lendable > 0 {
-			ratio = max(rStats.Usage+wUsage[rName]-rStats.Nominal, 0) * 100 /
+			ratio = max(rStats.Usage+wUsage[rName]*m-rStats.Nominal, 0) * 100 /
```

Comment:
Here (`wUsage`), a nil map could happen. I'm sure that any nil pointer error will never happen here even if the `wUsage` is nil.
But, please let me confirm if there are not any other potential errors.

Reply:
This is not a problem https://go.dev/play/p/pP1v9ALY6hh

Reply:
Thank you for showing that!

Reply:
/lgtm
a